Python-packages/covidcast-py/covidcast/covidcast.py

```diff
@@ -101,45 +101,57 @@ def signal(data_source: str,
       columns:
 
       ``geo_value``
-        identifies the location, such as a state name or county FIPS code. The
+        Identifies the location, such as a state name or county FIPS code. The
         geographic coding used by COVIDcast is described in the `API
         documentation here
         <https://cmu-delphi.github.io/delphi-epidata/api/covidcast_geography.html>`_.
 
+      ``signal``
+        Name of the signal, same as the value of the ``signal`` input argument. Used for
+        downstream functions to recognize where this signal is from.
+
       ``time_value``
-        contains a `pandas Timestamp object
+        Contains a `pandas Timestamp object
         <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Timestamp.html>`_
         identifying the date this estimate is for.
 
+      ``direction``
+        Uses a local linear fit to estimate whether the signal in this region is
+        currently increasing or decreasing (reported as -1 for decreasing, 1 for
+        increasing, and 0 for neither).
+
```

Comment:
Direction hasn't been updated in any signal since August; how does the API interpret that? NA?